tests/baselines/reference/contextualTypeWithUnionTypeObjectLiteral.errors.txt

```diff
@@ -22,9 +22,9 @@ tests/cases/conformance/types/union/contextualTypeWithUnionTypeObjectLiteral.ts(
       Type 'string | number' is not assignable to type 'number'.
         Type 'string' is not assignable to type 'number'.
 tests/cases/conformance/types/union/contextualTypeWithUnionTypeObjectLiteral.ts(58,5): error TS2322: Type '(a: string, b: number) => string | number' is not assignable to type '((a: string, b: number) => string) | ((a: string, b: number) => number)'.
-  Type '(a: string, b: number) => string | number' is not assignable to type '(a: string, b: number) => number'.
-    Type 'string | number' is not assignable to type 'number'.
-      Type 'string' is not assignable to type 'number'.
+  Type '(a: string, b: number) => string | number' is not assignable to type '(a: string, b: number) => string'.

```

Comment:
We could avoid these baseline changes if we choose the last matching invokable instead of the first.